test/e2e/performanceprofile/functests/13_llc/llc.go

```diff
@@ -234,12 +237,23 @@ var _ = Describe("[rfe_id:77446] LLC-aware cpu pinning", Label(string(label.Open
 					By(fmt.Sprintf("Waiting when %s finishes updates", poolName))
 					profilesupdate.WaitForTuningUpdated(ctx, perfProfile)
 				}
-
-				for _, node := range workerRTNodes {
-					kubeletconfig, err := nodes.GetKubeletConfig(ctx, &node)
-					Expect(err).ToNot(HaveOccurred())
-					Expect(kubeletconfig.CPUManagerPolicyOptions).ToNot(HaveKey("prefer-align-cpus-by-uncorecache"))
-				}
+				// Use Eventually in case of any delays in updating kubelet
+				Eventually(func() bool {
+					for _, node := range workerRTNodes {
+						kubeletconfig, err := nodes.GetKubeletConfig(ctx, &node)
+						if err != nil {
```

Comment:
I think that here we should abort. 
If we're failing the get kubelet from the API server, it means wer'e in a pretty bad shape